src/basic_recipes/tricontourf.jl

```diff
@@ -0,0 +1,198 @@
+- `triangulation = DelaunayTriangulation()`. The mode with which the points in `xs` and `ys` are triangulated. Passing `DelaunayTriangulation()` performs a delaunay triangulation. You can also pass a preexisting triangulation as an `AbstractMatrix{<:Int}` with size (3, n), where each column specifies the vertex indices of one triangle.
+
+### Generic
+
+- `visible::Bool = true` sets whether the plot will be rendered or not.
+- `overdraw::Bool = false` sets whether the plot will draw over other plots. This specifically means ignoring depth checks in GL backends.
+- `transparency::Bool = false` adjusts how the plot deals with transparency. In GLMakie `transparency = true` results in using Order Independent Transparency.
+- `fxaa::Bool = false` adjusts whether the plot is rendered with fxaa (anti-aliasing).
+- `inspectable::Bool = true` sets whether this plot should be seen by `DataInspector`.
+- `depth_shift::Float32 = 0f0` adjusts the depth value of a plot after all other transformations, i.e. in clip space, where `0 <= depth <= 1`. This only applies to GLMakie and WGLMakie and can be used to adjust render order (like a tunable overdraw).
+- `model::Makie.Mat4f` sets a model matrix for the plot. This replaces adjustments made with `translate!`, `rotate!` and `scale!`.
+- `color` sets the color of the plot. It can be given as a named color `Symbol` or a `Colors.Colorant`. Transparency can be included either directly as an alpha value in the `Colorant` or as an additional float in a tuple `(color, alpha)`. The color can also be set for each scattered marker by passing a `Vector` of colors or be used to index the `colormap` by passing a `Real` number or `Vector{<: Real}`.
+- `colormap::Union{Symbol, Vector{<:Colorant}} = :viridis` sets the colormap from which the band colors are sampled.
+
+## Attributes
+$(ATTRIBUTES)
+"""
+@recipe(Tricontourf) do scene
+    Theme(
+        levels = 10,
+        mode = :normal,
+        colormap = theme(scene, :colormap),
+        extendlow = nothing,
+        extendhigh = nothing,
+        nan_color = :transparent,
+        inspectable = theme(scene, :inspectable),
+        transparency = false,
+        triangulation = DelaunayTriangulation()
+    )
+end
+
+function Makie.convert_arguments(::Type{<:Tricontourf}, x::AbstractVector{<:Real}, y::AbstractVector{<:Real}, z::AbstractVector{<:Real})
+    map(x -> elconvert(Float32, x), (x, y, z))
+end
+
+
+function Makie.plot!(c::Tricontourf)
+    xs, ys, zs = c[1:3]
+
+    c.attributes[:_computed_levels] = lift(zs, c.levels, c.mode) do zs, levels, mode
+        _get_isoband_levels(Val(mode), levels, vec(zs))
+    end
+
+    colorrange = lift(c._computed_levels) do levels
+        minimum(levels), maximum(levels)
+    end
+    computed_colormap = lift(c._computed_levels, c.colormap, c.extendlow,
```

Comment:
should we share this with contourf?

Reply:
Yeah that's copied, should be factored out